src/index.html

```diff
@@ -11,7 +11,124 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST"
+    onsubmit="onSubmit()">
+      <fieldset>
+        <legend>
+          Personal information
+        </legend>
+        <div class="form-field">
+          <label for="surname">Surname:</label>
+          <input id="surname" type="text"
+          name="surname" autocomplete="off">
+        </div>
+        <div class="form-field">
+          <label for="name">Name:</label>
+          <input id="name" type="text"
+          name="name" autocomplete="off">
+        </div>
+        <div class="form-field">
+          <label for="age">How old are You?</label>
+          <input id="age" type="number"
+          name="age" value="12"
+          min="7" max="100">
+        </div>
+        <div class="form-field">
+          <label for="birth">Full date of birth:</label>
+          <input id="birth" type="date"
+          name="date of birth">
+        </div>
+        <div class="form-field">
+          <label for="terms">I accept the term of the agreement</label>
+          <input id="terms" type="checkbox"
+          name="terms">
+        </div>
+      </fieldset>
+      <fieldset>
+        <legend>Registration</legend>
+        <div class="form-field">
+          <label for="email">E-mail:</label>
+          <input id="email" type="email"
+          name="email" placeholder="email@example.com">
+        </div>
+        <div class="form-field">
+          <label for="password">Password:</label>
+          <input type="password" name="password"
+          id="password" minlength="3"
+          maxlength="15">
+        </div>
+      </fieldset>
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          Do you love cats?
+          <label for="yes">
+            <input type="radio" name="yes"
+            id="yes">
+            Yes
+          </label>
+          <label for="no">
+            <input type="radio" name="yes"
+            id="no">
+            No
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="color">
+            What is your favorite color?
+            <input type="color" name="color"
+            id="color">
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="bed-time">
+            What time do you go to bed?
+            <input type="time" id="bed-time"
+            name="bed-time">
```

Comment:
Use camelCase for `id` and `name`
```suggestion
            <input type="time" id="bedTime"
            name="bedTime">
```